tree.js

```diff
@@ -0,0 +1,191 @@
+/* eslint-disable import/prefer-default-export */

```

Comment:
Avoid disabling eslint rules. Figure out why lint errors are coming and explore and make necessary changes.

Reply:
Implemented the changes